bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+
+The following sections refer to BIP Status Field values. The BIP Status Field is defined in the Header Preamble
+specification above.
+
+#### Draft
+
+After fleshing out the proposal further and ensuring that it is of high quality and properly formatted, the author should
+open a pull request to the [BIPs repository](https://github.com/bitcoin/bips). The document must adhere to the
+formatting requirements specified below and be named with a working title of the form "bip-title". The author must not
+self-assign a number to their proposal.
+
+BIPs that (1) adhere to the formatting requirements, (2) are on-topic, and (3) have materially progressed
+beyond the ideation phase, e.g. by generating substantial public discussion and commentary from diverse contributors, by
+independent bitcoin projects working on adopting the proposal, or by the author working for an extended period
+toward improving the proposal based on community feedback, will be assigned a number by a BIP editor. The BIP editors
+should delay number assignment when they perceive a proposal being met with lack of interest: number
+assignment facilitates the distributed discussion of ideas, but before a proposal garners some interest in the bitcoin
+community, there is no need to refer to it by a number.
+
+Proposals are also not ready for number assignment if they duplicate efforts, disregard formatting rules, are too
+unfocused or too broad, fail to provide proper motivation, fail to address backwards compatibility, or fail to specify
+the feature clearly and completely. Reviewers and BIP editors should provide guidance on how the proposal may
+be improved to progress toward readiness. Pull requests that are proposing off-topic or unserious ideas or have
+stopped to make progress may be closed.
+
+When the proposal has been assigned a number and is complete, a BIP editor will merge it to the BIPs repository.
+After the BIP is merged to the repository, its main focus should no longer shift significantly, even while the author
+may continue to update the proposal as necessary in the git repository. Updates to merged documents by the author should also be
+submitted as pull requests.
+
+#### Proposed[^proposed]
+
+When the author is confident that their BIP represents a net improvement, is clear, comprehensive, has a working
+reference implementation (if applicable), and is ready for adoption by the bitcoin community, they may update the BIP’s
+status to Proposed to indicate that they recommend adoption or implementation of the BIP. Where applicable, the
+author must ensure that any proposed specification is solid, not unduly complicated, and definitive. Subsequently, the
+BIP’s content should only be adjusted in minor details, e.g. to improve language, clarify ambiguities, backfill
+omissions in the specification, add test vectors for edge cases, or address other issues discovered as the BIP is
+adopted.
```

Comment:
Hm, I haven't thought this through entirely, but perhaps this rule is a bit too strict. I think there could be exceptions, e.g., if a bug or vulnerability is found later etc. Perhaps the changes won't be minor details. Perhaps the use of the word "should" expresses the ability for exceptions, but then it may be better to make this more explicit.

style nit:
```suggestion
author must ensure that any proposed specification is solid, not unduly complicated, and definitive. Subsequently, the
BIP’s content should only be adjusted in minor details, e.g. to improve language, clarify ambiguities, backfill
omissions in the specification, add test vectors for edge cases, or address other issues discovered as the BIP is
being adopted.
```

Reply:
Agreed.